test/regressions/TestViewer.js

```diff
@@ -86,7 +86,7 @@ function TestViewer(props) {
           <Box
             aria-busy={!ready}
             data-testid="testcase"
-            sx={{ bgcolor: 'background.default', display: 'inline-block', p: 1 }}
+            sx={{ bgcolor: 'background.default', display: 'flex', p: 1 }}
```

Comment:
I wonder if we should set the exact same styles as in the demos (source: https://github.com/mui/material-ui/blob/v6.1.0/docs/src/modules/components/Demo.js#L216-L219), namely add `position: 'relative'` and `justifyContent: 'center'`.